src/Library/demos/Text Input/main.blp

```diff
@@ -0,0 +1,309 @@
+using Gtk 4.0;
+using Adw 1;
+
+Adw.Clamp {
+  Box {
+    orientation: vertical;
+
+    Label {
+      label: "Text Input";
+      margin-top: 10;
+      margin-bottom: 10;
+
+      styles [
+        "title-1",
+      ]
+    }
+
+    Label {
+      label: "Widgets used for reading text input";
+      margin-bottom: 10;
+    }
+
+    Label {
+      label: "Entry";
+      margin-top: 10;
+      halign: start;
+
+      styles [
+        "heading",
+      ]
+    }
+
+    Label {
+      label: "A simple but versatile widget that allows single line text entry and editing.";
+      margin-top: 10;
+      margin-bottom: 10;
+      halign: start;
+
+      styles [
+        "dim-label",
+      ]
+    }
+
+    FlowBox {
+      homogeneous: true;
+      row-spacing: 20;
+
+      Box {
+        orientation: vertical;
+
+        Label {
+          label: "Regular";
+          margin-bottom: 10;
+        }
+
+        Entry entry {
+
+        }
+      }
+
+      Box {
+        orientation: vertical;
+
+        Label {
+          label: "Overwritable";
+          margin-bottom: 10;
+        }
+
+        Entry {
+          overwrite-mode: true;
```

Comment:
This is too rarely used to showcase

You can remove it

Reply:
👍🏾